Languages/update_android_source_strings.py

```diff
@@ -0,0 +1,50 @@
+from pathlib import Path
+from lxml import etree
+import os
+import polib
+
+# Remove <string> tags from each string
+# Normally when getting the body text, this would not include inner tags.
+# This becomes a problem when parsing strings with clickable links.
+# So instead of manually breaking the body down and reassembling it,
+# here we just do some string manipulation to save some time.
+def get_xml_string(xml_string):
+    item = etree.tostring(xml_string, encoding='unicode')
+    end_first_tag = item.index('>')
+    first_half = item[end_first_tag+1:]
+    start_second_tag = first_half.rindex('<')
+    output = first_half[:start_second_tag]
+    return output.replace('\\n', '\n').replace("\\'", "'")
```

Comment:
There are other characters that need the same treatment. `"`, for instance.

Reply:
`"` is already formatted correctly since qt does the same thing right?

Reply:
And to be clear, I would prefer a comprehensive solution here, not just that we fix everything we see popping up in the resulting .pot file.

Reply:
That's entirely reasonable. I just don't think I know all of the characters that should be escaped by `\`
I'll look around